src/ui/frames.c

```diff
@@ -98,6 +98,23 @@ meta_frames_constructor (GType                  gtype,
   return object;
 }
 
+static void
+resolution_updated (MetaFrames *frames)
+{
+  gtk_style_context_invalidate (gtk_widget_get_style_context (GTK_WIDGET (frames)));
```

Comment:
Might be nice to work out why this is needed (various games?) & perhaps upstream it.

Reply:
Sure. See https://phabricator.endlessm.com/T26405